### PR TITLE
Define the initial user guides layout

### PR DESCRIPTION
**Pull request by andyrogers1973**
This PR 'defines' the proposed layout of the user guides site:

- added .html.md.erb files to create the structure
- amended index.html.md.erb to generate the landing page outlining this layout
- corrected a few typo's

All the pages have been left blank deliberately - consolidating and updating the information contained in the pages will happen once the layout is agreed and in place.

### CODE REVIEW

File: source/documentation/information/mojgithubenteprise.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: MoJ GitHub Enteprise Management
+last_reviewed_on: 2023-08-02
+review_in: 6 months
+---
+
+# MoJ GitHub Enteprise Management
```
Comment (NickWalt01) [inline source/documentation/information/mojgithubenteprise.html.md.erb:8]: typo: enterprise and in the filename as well

File: source/index.html.md.erb [hunk 2]
```diff
@@ -1,19 +1,44 @@
...
 
-These guides provide infomration and details of the services that Operations Engineering support.
+These guides provide information and details about the services that Operations Engineering support.
 
 ## Overview
 
-[Who are Operations Engineering?](documentation/information/operations-engineering.html)
-[Our Allinace](documentation/information/our-alliance.html)
+*[Who are Operations Engineering?](documentation/information/operations-engineering.html)
+*[MoJ GitHub Enteprise Management](documentation/information/mojgithubenteprise.html)
```
Comment (NickWalt01) [inline source/index.html.md.erb:15]: typo: enterprise

Comment (andyrogers1973) [reply source/index.html.md.erb]: All fixed now

File: source/documentation/information/mojgithubenteprise.html.md.erb (hunk 1 above)
Comment (andyrogers1973) [reply source/documentation/information/mojgithubenteprise.html.md.erb:8]: fixed

**Review by ben-al — APPROVED** [review]

**Pull request merged**